internal/landing/html/inspect.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title> Inspect debug data </title>
+    </head>
+    <body>
+        <form class="form-group" action="/inspect" method="POST">
```

Comment:
it might be handy to make form use `GET` method (a bit different to query args via URL), but this way one could build a URL like `https://discovery.talos.dev/inspect?clusterID=<abcd>` to show the information.

As this is read-only page, `POST` is not required.